datahub-web-react/src/app/ingest/source/executions/ExecutionRequestDetailsModal.tsx

```diff
@@ -161,14 +182,32 @@ export const ExecutionDetailsModal = ({ urn, visible, onClose }: Props) => {
                         </Button>
                     </SectionSubHeader>
                     <Typography.Paragraph ellipsis>
-                        <pre>{`${logs}${!showExpandedLogs && isOutputExpandable ? '...' : ''}`}</pre>
-                        {isOutputExpandable && (
+                        <pre>{`${logs}${!showExpandedLogs && areLogsExpandable ? '...' : ''}`}</pre>
+                        {areLogsExpandable && (
                             <ShowMoreButton type="link" onClick={() => setShowExpandedLogs(!showExpandedLogs)}>
                                 {showExpandedLogs ? 'Hide' : 'Show More'}
                             </ShowMoreButton>
                         )}
                     </Typography.Paragraph>
                 </LogsSection>
+                {recipe && (
+                    <RecipeSection>
+                        <SectionHeader level={5}>Recipe</SectionHeader>
```

Comment:
I feel like this whole recipe should be hidden by default and expanded only if they care to see it, as opposed to showing a truncated recipe with `...` inside of the code block for the recipe itself that you show more or less of - which feels weird itself tbh (even though we do that with logs above it)

Reply:
maybe we can make it more like the "Advanced" section at the end of defining a recipe with a toggle button to the side of the title "Recipe" (which could remain the same styling as you have now) that would show or hide the whole recipe section, so users who don't care about the recipe don't see it at all

<img width="949" alt="image" src="https://github.com/datahub-project/datahub/assets/28656603/b9216ebc-dec8-4490-9c81-2ebc807d7989">

Reply:
alternatively to hiding the whole section, I'd like to show less of the recipe so that it doesn't show so much right away like i'm seeing in your screenshot (seems like we show 75% of the recipe right away and takes up much more room than the logs part)

Reply:
I'm happy to do either, just not sure the best way to do the styling... like the `> Advanced` is pretty clear it's expandable but how to do the same for the recipe section lol.

I am showing exactly 10 lines of the recipe right now. I'll just drop that down to 1-3

Reply:
<img width="779" alt="Screenshot 2023-10-17 at 7 11 28 PM" src="https://github.com/datahub-project/datahub/assets/5423622/17540b8e-ba80-4674-be13-c55592fb67b5">
This good?

Reply:
much better!